tests/components/shelly/__init__.py

```diff
@@ -63,3 +74,44 @@ def inject_rpc_device_event(
     """Inject event for rpc device."""
     monkeypatch.setattr(mock_rpc_device, "event", event)
     mock_rpc_device.mock_event()
+
+
+async def mock_rest_update(hass: HomeAssistant):
+    """Move time to create REST sensors update event."""
+    async_fire_time_changed(
+        hass, dt.utcnow() + timedelta(seconds=REST_SENSORS_UPDATE_INTERVAL)
+    )
+    await hass.async_block_till_done()
+
+
+def create_entity(
+    hass: HomeAssistant,
+    domain: str,
+    object_id: str,
+    unique_id: str,
+    config_entry: ConfigEntry | None = None,
+) -> str:
+    """Create enabled entity, return entity_id."""
+    entity_registry = async_get(hass)
+    entity_registry.async_get_or_create(
+        domain,
+        DOMAIN,
+        f"{MOCK_MAC}-{unique_id}",
+        suggested_object_id=object_id,
+        disabled_by=None,
+        config_entry=config_entry,
+    )
+    return f"{domain}.{object_id}"
+
+
+def create_device(device_reg, config_entry: ConfigEntry):
```

Comment:
I'd probably name this `register_entity` and `register_device` so its clear we are not actually accessing the entity object here since someone may copy these tests in the future and get the wrong idea.

Reply:
Register explain this much better 👍